router/batchrouter/batchrouter.go

```diff
@@ -826,6 +829,38 @@ func (brt *HandleT) asyncUploadWorker(ctx context.Context) {
 	}
 }
 
+func resolveURL(base, relative string) string {
+	baseURL, err := url.Parse(base)
+	if err != nil {
+		pkgLogger.Fatal(err)
+	}
+	relURL, err := url.Parse(relative)
+	if err != nil {
+		pkgLogger.Fatal(err)
+	}
+	destURL := baseURL.ResolveReference(relURL).String()
+	return destURL
+}
+
+func (brt *HandleT) parseUploadIntervalFromConfig(destinationConfig map[string]interface{}) time.Duration {
+	uploadInterval, ok := destinationConfig["uploadInterval"]
+	if !ok {
+		brt.logger.Debugf("BRT: uploadInterval not found in destination config, falling back to default: %s", brt.asyncUploadTimeout)
+		return brt.asyncUploadTimeout
+	}
+	dur, ok := uploadInterval.(string)
+	if !ok {
+		brt.logger.Debugf("BRT: not found string type uploadInterval, falling back to default: %s", brt.asyncUploadTimeout)
```

Comment:
I would use a warn here since a type other than a string would be problematic:
```suggestion
		brt.logger.Warnf("BRT: not found string type uploadInterval, falling back to default: %s", brt.asyncUploadTimeout)
```

Reply:
How often this is called, could it pollute the log files with this?